### PR TITLE
Feature/CV2-5204 - Display article card always when list is returning, articles

### PR DESCRIPTION
**Pull request by sarahkeh**
## Description

Replaced SmallMediaCard with ArticleCard. By passing a fact-check or media value down to the search component, the component can determine which card type to render. Small optional props were added to the ArticleCard for this: a classNames prop in order to pass it the Selected styling, and a readOnly boolean for the tags.  From the search, I added several fact_check and claim_description fields that were needed for the Card, as well as the team's verification_statuses. 

I set the Tags component to ReadOnly as it didn't seem like an appropriate time for the user to make tag changes to a Fact-Check.  It also would involve mixing Relay and Fetch in one component which felt not great. 

Screenshot 2024-10-15 at 1 39 30 AM

References: CV2-5204

## Type of change

- [X] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

manually, checking both that the media cards in the other tabs were not impacted and the article card matched the card in the fact-checks list

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

Tags as Read-Only - this felt like the simpler user decision but I can change this easily if needed

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)
- [ ] If I touched a file that included an eslint-disable-file header, I updated the code such that the disabler can be removed

### CODE REVIEW

**Review by brianfleming — COMMENTED** [review]
@sarahkeh this is looking good. I will let others comment on the tech for actual approvals.
But I had one request, now that I am seeing this in real life, something to add to this PR that is kind of related.
Overtime the Tags List has changed, but I think we should make one more change to have this interaction a bit more clear:
- When the tag list is read only (like in this use case) can we hide the "manage tags" button instead of just disabling?
- I don't think there is value in seeing that button if you can't interact with it.

thoughts?

**Review by danielevalverde — COMMENTED** [review]
:lgtm:

**Comment by sarahkeh** [comment]
> @sarahkeh this is looking good. I will let others comment on the tech for actual approvals. But I had one request, now that I am seeing this in real life, something to add to this PR that is kind of related. Overtime the Tags List has changed, but I think we should make one more change to have this interaction a bit more clear:
> 
> * When the tag list is read only (like in this use case) can we hide the "manage tags" button instead of just disabling?
> * I don't think there is value in seeing that button if you can't interact with it.
> 
> thoughts?

I had that thought, too.  Just pushed the change, makes more sense than an unused button 

Screenshot 2024-10-15 at 10 58 25 AM

File: src/app/components/cds/menus-lists-dialogs/TagPicker.js [hunk 1]
```diff
@@ -124,29 +124,31 @@ const TagPicker = ({
...
+              defaultMessage="Manage Tags"
+              description="Tooltip message displayed on a tag item to let the user know they can manage the tags in the list"
+              id="taglist.tooltipManage"
+            />
+          }
+        >
+          <div>
+            <ButtonMain
+              className="int-tag-list__button--manage"
+              disabled={readOnly}
```
Comment (amoedoamorim) [inline src/app/components/cds/menus-lists-dialogs/TagPicker.js:142]: `disabled={readOnly}` should be removed now that the button isn't rendered

**Review by amoedoamorim — APPROVED** [review]

**Pull request merged**